### ISSUE TITLE
OnHover magically appearing links

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/sketchup-yard-template#14]
Thomas made note in another issue thread about liking these style links:
http://breathe.readthedocs.io/en/latest/#_CPPv2N8NutshellD0Ev

Basically they are onHover (onMouseOver) magically appearing links.
They'd work well for the permalinks.

I couldn't find the original thread in which Thomas mentioned this, so I started a new one to talk about **link styling**.

**Comment by jimfoltz** [comment]
> I couldn't find the original thread 

I'm honesty having a hard time following any and all the discussions going on the the issues. I have to go through several issues in several repos to find what I wanted to review.

**Comment by DanRathbun** [comment]
**This thread is redirected to discuss confusion over discussions (because like the other discussions, it was yanked offtopic!)**

**Comment by DanRathbun** [comment]
Yes, Jim, it can be confusing, because GitHub gives us so many places to comment.

**1.** 1st problem is the HTML "docs" repo. This repo doesn't do anyone much good at all. (Unless they wish to just copy all the pages to their local drive. But, then the pages could be packaged up in a zip.)
But this was before the stubs were setup to be the "working repo" for the API documentation. So most the doc bug issues there need to be migrated to the **ruby-api-stubs** repo. (And then the "docs" repo removed.)

**2.** Anything** discussing, or promoting ideas for the styling and functionality of the `http://ruby.sketchup.com/` needs to be migrated to the **sketchup-yard-template** repo.

**3.** Then issues and discussions need to be put and **stay**, in the proper repo.

**4.** Replying to comments, issues, pull requests, etc., ... Need to stay on topic. 
Otherwise start a new topic, or issue. This is especially true for issues and PRs.

(Everyone so far involved has been guilty of meandering offtopic, I as well.)

**Comment by thomthom** [comment]
> 1st problem is the HTML "docs" repo. This repo doesn't do anyone much good at all.

That's actually where the Ruby API docs are served from, we are using the GitHub pages platform.

What we can do is set up Issue tracker template that contain information and guidance to what issue should be filed where; https://github.com/blog/2111-issue-and-pull-request-templates

Everything right now is rather new, but as we work out the format of all this things will settle down.

**Comment by DanRathbun** [comment]
So, whilst on the topic of "offtopic confusion", I asked nicely if we could cleanup the offtopic noise you (Jim) created in PR#10. Ie:
https://github.com/SketchUp/ruby-api-stubs/pull/10#issuecomment-273950720

**We cannot be doing this in PRs!** A PR needs to be requested against a specific Issue or Issues. It then needs to stand up to the test of whether it satisfies these open issues, and NOTHING MORE.

Otherwise, we never get anything merged! And Issues will grow and grow and grow, and never end.

The best Issues and PRs are small concise ones. These are ones that Thomas will be able to REVIEW quickly, and get merged. The bigger they are, the longer it will take, the more likely he'll get interrupted and lose his place and train of thought. (And probably have to begin again, later.)

This is _like_ basic Engineering 101 - Drawing Change protocol.

The GitHub Issue is a like a DCR. Assignment to the drafter by the Documentation Manager. Pull Request is like a request by the drafter (to the Checker) to publish the changes they've made. The Merge is like the distribution of an ECN with prints, by the Configuration Manager.

It is not allowed to add NEW changes in at step 4 by the Checker, if they are not on the DCR. (Otherwise the whole process would have to stop and return back to the engineer who originated the DCR. Then resubmittal to the Doc. Mgr., reassignment to the drafter, request again to the checker, loop dee-doopdy.)

**Comment by DanRathbun** [comment]
> I asked nicely if we could cleanup the offtopic noise you (Jim) created in PR#10.

**Never mind I redacted PR#10.** (I didn't put in it it's own branch, nor the next edit, and they started appending to one another, because both were form master.)

**Comment by DanRathbun** [comment]
> @jimfoltz: I'm honesty having a hard time following any and all the discussions going on the the issues. I have to go through several issues in several repos to find what I wanted to review.

Ah Ha! I learned something a min ago, by accident.

Go to your GitHub homepage.
Scroll down to the **Contribution activity** section,
Beneath this are expandable sections with expandable subsections.

**Comment by DanRathbun** [comment]
@jimfoltz And here's another way of getting at "your stuff"

Click the bell icon at the upper right of any page. This is the notifications page.
You can filter by "Unread", "Participating" and "All" 
When "All" is selected, you'll see each repo in which you're participating in the content pane.
(You can reduce this by clicking the individual repo filters in the left column (which shows the tally of issues and Pulls, etc., you're involved with.)

You can also switch to the "Watching" tab, to get at the list of all repos you've clicked the "Watch" button on (anywhere on GitHub.)

**Comment by jimfoltz** [comment]
Thanks @DanRathbun, that's helpful.  The other problem is staying on topic...  some things I wanted to look  up have nothing to do with the title of the issue.  I go off-topic too often myself.

**Comment by DanRathbun** [comment]
Currently stuff comes into Gmail, like chained convos, that are missing any post I first view on GitHub. So they are not complete chains. And can be confusing.

I'd be better to just get a "busy" post, like a tweet that I have notifications, rather than a bunch of separate emails.
